packages/lesswrong/components/common/Header.tsx

```diff
@@ -256,6 +270,13 @@ const Header = ({standaloneNavigationPresent, toggleStandaloneNavigation, stayAt
     SearchBar, UsersMenu, UsersAccountMenu, NotificationsMenuButton, NavigationDrawer,
     NotificationsMenu, KarmaChangeNotifier, HeaderSubtitle, Typography
   } = Components;
+  
+  const usersMenuClass = isEAForum ? classes.hideXsDown : classes.hideMdDown
+  const usersMenuNode = currentUser && <div className={searchOpen ? usersMenuClass : undefined}>
+    <AnalyticsContext pageSectionContext="usersMenu">
+      <UsersMenu />
+    </AnalyticsContext>
+  </div>
```

Comment:
I also updated how the EA Forum handles the search box in the header. We now only hide the other icons on **extra small** screens, and we'll hide **all** the other icons rather than just the user menu button.